Microsoft.Toolkit.Uwp.UI/AdvancedCollectionViewSource/ICollectionViewEx.cs

```diff
@@ -0,0 +1,73 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Windows.UI.Xaml.Data;
+
+namespace Microsoft.Toolkit.Uwp.UI
+{
+    /// <summary>
+    /// Extended ICollectionView with filtering and sorting
+    /// </summary>
+    public interface ICollectionViewEx : ICollectionView
```

Comment:
Why not call it "IAdvancedCollectionView" to match the class name.
Or "ICollectionViewSource" if the class name is named "CollectionViewSource"